index.html

```diff
@@ -23,97 +27,110 @@
             <div class='nav-link'>
                 <a href="#contacts">Hit me up!</a>
             </div>
+            <div class="nav-hamburger">
+                <img src="" alt="">
+            </div>
+            <div class="nav-cross">
+                <img src="images/burger.png" alt="a menu icon">
+            </div>
         </div>
     </nav>
     <main>
         <div id='bio'>
             <div class='bio-title'>My Story so far:</div>
             <div class='bio-container'>
                 <div class='bio-text'>
-                    <h1>Yo! I'm Rich Herbert <br/>aka. RH Design</h1>
-                    <p>I'm a full-stack developer. I've studied a degree in computer science and am currently training at the Mayden Academy. I have a passion for agile methodologies, holistic development and clean, reuseable code.</p>
+                    <div class='bio-intro'>Yo! I'm Rich Herbert <br/>aka. RH Design</div>
+                    <div class='bio-story'>I'm a full-stack developer and currently studying a degree in computer science and am currently training at the Mayden Academy. I have a passion for agile methodologies, holistic development and clean, reuseable code.</div>
                 </div>
-                <div class='bio-pic'></div>
+                <img class="bio-pic" src="images/rich.jpg" alt="">
```

Comment:
Double quotes here, singles quotes elsewhere. Just change to all be the same, doesn't matter which!